#!/usr/bin/env python
PACKAGE = "project1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

int_enum = gen.enum([ gen.const("Euler",      int_t, 0, "Euler integration"),
                      gen.const("RungeKutta", int_t, 1, "Runge-Kutta integration") ],
                      "Numerical integration method")

gen.add("intMethod", int_t, 0, "Integration method for odometry computation", 1, 0, 1, edit_method=int_enum)

exit(gen.generate(PACKAGE, "robot", "IntegrationMethod"))